src/gui/widgets/overlay/subtitlewidget.h

```diff
@@ -212,7 +212,15 @@ private Q_SLOTS:
 
         /* true if subtitles should be shown when needed, false otherwise. */
         bool showSubtitles;
+
+	/* True if playback should be paused for the current subtitle, false
+	 * otherwise. */
+	bool pauseOnSubtitleEnd;
     } m_settings;
+
+    /* True if playback has already been paused for the current subtitle, false
+     * otherwise. */
+    bool pausedForCurrentSubtitle = false;
```

Comment:
This is a member variable so be sure to prefix it with `m_`.
Also move it up below `m_paused` since it isn't a inside a struct.